packages/frontend-shared/src/locales/en-US.json

```
@@ -520,6 +521,10 @@
     "component": {
       "name": "Component Testing",
       "description": "Build and test your components from your design system in isolation in order to ensure each state matches your expectations."
+    },
+    "componentDisabled": {
+      "description": "Testing your components requires {0} as an NPM dependency for this project.",
+      "link": "installing Cypress"
```

I'd prefer active over passive voice here.

```suggestion
      "description": "To test your components you must {0} as an NPM dependency for this project.",
      "link": "install Cypress"
```

---

@BlueWinds going with your suggestion